projects/components/src/navigation/nav-item/nav-item.component.ts

```diff
@@ -57,12 +57,23 @@ export class NavItemComponent {
   @Input()
   public readonly navItemViewStyle?: NavViewStyle;
 
-  public buildNavigationParam = (item: NavItemLinkConfig): NavigationParams => ({
-    navType: NavigationParamsType.InApp,
-    path: item.matchPaths[0],
-    relativeTo: this.activatedRoute,
-    replaceCurrentHistory: item.replaceCurrentHistory
-  });
+  public buildNavigationParam = (item: NavItemLinkConfig): NavigationParams => {
+    const timeRange = this.config.timeRangeResolver ? this.config.timeRangeResolver() : undefined;
```

Comment:
nit - optional chain
```suggestion
    const timeRange = this.config.timeRangeResolver?.();
```